### ISSUE TITLE
🐛 `assert` keyword in import causes issue when using rome v12.1.0

### ISSUE THREAD

**Issue by Aeolun** [rome/tools#4485]
### Environment information

```block
CLI:
  Version:                      12.1.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           macos

Environment:
  ROME_LOG_DIR:                 unset
  NO_COLOR:                     unset
  TERM:                         "xterm-kitty"
  JS_RUNTIME_VERSION:           "v18.16.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "null"

Rome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    true
  VCS disabled:                 true

Workspace:
  Open Documents:               0

Discovering running Rome servers...
```

### What happened?

We run `rome format .`. Rome 12.0.0 is perfectly happy, but rome 12.1.0 outputs:

```
  ✖ Expected a semicolon or an implicit semicolon after a statement, but found none
  
    3 │ import ValidationError from 'errorhandler';
    4 │ import queuer from './logic/file.js';
  > 5 │ import submitOrder from './schemas/queuer-request.json' assert { type: 'json' };
      │                                                                ^
    6 │ 
    7 │ const ajv = new Ajv({
  
  ℹ An explicit or implicit semicolon is expected here...
  
    3 │ import ValidationError from 'errorhandler';
    4 │ import queuer from './logic/file.js';
  > 5 │ import submitOrder from './schemas/queuer-request.json' assert { type: 'json' };
      │                                                                ^
    6 │ 
    7 │ const ajv = new Ajv({
  
  ℹ ...Which is required to end this statement
  
    3 │ import ValidationError from 'errorhandler';
    4 │ import queuer from './logic/file.js';
  > 5 │ import submitOrder from './schemas/queuer-request.json' assert { type: 'json' };
      │                                                         ^^^^^^^^
    6 │ 
    7 │ const ajv = new Ajv({
```

### Expected result

I expect the formatting to finish with no errors.

### Code of Conduct

- [X] I agree to follow Rome's Code of Conduct

**Comment by nstepien** [comment]
I have the same issue.

For additional context:
Rome added support for the [`with` import attributes](https://github.com/tc39/proposal-import-attributes) grammar, but no JS engine support it yet.
Not all tools support it either, notably TypeScript supports `assert` but not `with`.
I think Rome should support both as long as the `with` grammar is not a viable replacement in the ecosystem.

**Comment by Conaclos** [comment]
The [spec](https://github.com/tc39/proposal-import-attributes) has changed. Rome updated to support the new syntax: we need to use `with` instead of `assert`.

**Comment by ematipico** [comment]
We were too fast to implement the new grammar 😅 

Theoretically, we could resurrect the old syntax quickly and push it as a fix.

**Comment by nstepien** [comment]
>we could resurrect the old syntax quickly and push it as a fix.

I'd love that.
`with` grammar support is great, but yeah it's just too early to replace `assert`.

**Comment by Aeolun** [comment]
Regardless of whether the reasons make sense, it would be nice if support for specific things wasn't removed in minor version updates.

**Comment by nstepien** [comment]
@ematipico should we reopen this issue until the `assert` grammar is added back?

**Comment by ematipico** [comment]
Closing, this should be fixed in `v12.1.3`

**Comment by nstepien** [comment]
It works again, thanks!
